Xamarin.PropertyEditing.Tests/MockControls/MockSampleControl.cs

```diff
@@ -16,6 +16,7 @@ public MockSampleControl ()
 			AddProperty<int> ("UnsetInteger", ReadWrite, valueSources: ValueSources.Local);
 			AddProperty<float> ("FloatingPoint", ReadWrite);
 			AddProperty<string> ("String", ReadWrite, valueSources: ValueSources.Local | ValueSources.Resource | ValueSources.Binding);
+			AddProperty<int> ("Width", ReadWrite, valueSources: ValueSources.Local | ValueSources.Resource | ValueSources.Binding, inputModes: new[] { new InputMode("Auto", true), new InputMode("Star"), new InputMode("Pixel"), });
```

Comment:
Either those are going to surface to the user, and they need to be localizable, or they're magic strings, and could probably be replaced with an enum.

Reply:
They're up to the provider to provide. In theory they will likely correspond to fixed names from code (enums, const properties) that should not be translated.

Reply:
Ah yes, this is just the mock implementation, indeed.